src/platform/site-wide/announcements/components/PreDowntime.jsx

```diff
@@ -0,0 +1,48 @@
+// Dependencies.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import PromoBanner, {
+  PROMO_BANNER_TYPES,
+} from '@department-of-veterans-affairs/formation-react/PromoBanner';
+
+class PreDowntime extends Component {
+  static propTypes = {
+    announcement: PropTypes.shape({
+      downtimeStartsAt: PropTypes.string.isRequired,
+    }).isRequired,
+    dismiss: PropTypes.func.isRequired,
+  };
+
+  componentWillMount() {
+    // Set an interval to update the time.
+    this.rerenderInterval = setInterval(() => this.forceUpdate(), 60000);
+  }
```

Comment:
Do you think `forceUpdate` is better than setting an interval, then just updating the message in `state` to trigger the update?

Reply:
It's the same imo, I'm happy to do either! 🙂Long-term probably does make sense to just include it in state in case we build out state more with this component.